src/walletConnect/v1/saga.ts

```diff
@@ -207,7 +207,7 @@ function* acceptRequest(r: AcceptRequest) {
     SentryTransactionHub.finishTransaction(SentryTransaction.wallet_connect_transaction)
   } catch (e) {
     Logger.debug(TAG + '@acceptRequest', e.message)
-    connector?.rejectRequest({ id, jsonrpc, error: e.message })
+    yield put(denyRequestAction(peerId, request, e.message))
```

Comment:
At this point the action is not anymore in Redux. It's cleared when `ACCEPT_REQUEST_V1` is dispatched. And after that this saga is triggered.
So dispatching `denyRequest` does nothing and I think the previous behavior was good.

https://github.com/valora-inc/wallet/blob/0d6bff725be9f9ea634f6b7328fe318068bf55bc/src/walletConnect/v1/reducer.ts#L66-L71

Reply:
> So dispatching `denyRequest` does nothing and I think the previous behavior was good.

I meant in the Redux state. But it will be handled again by the saga. Which I think we shouldn't do as it will have other side effects, like capturing another analytics event for instance. Which is gonna be a bit misleading here. We shouldn't use the action for this case.

Reply:
yeah i think you're right, i came across a buggy looking scenario when i was testing the bottom sheets and thought this change fixed it but i cannot remember the exact details of the scenario anymore :( i'll close this PR for now and try to remember what i was trying to fix